File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/ADM101UgrdSystemItemInfo.java
package nta.med.data.model.ihis.adma;

public class ADM101UgrdSystemItemInfo {
	private String grpId      ;
	private String sysId      ;
	private String sysNm      ;
	private Double sysSeq     ;
	private String admSysYn  ;
	private String msgSysYn  ;
	private String sysDesc    ;
	private String mangDept   ;
	private String buseoName1  ;
	private String mangDept1  ;
	private String buseoName2  ;
	public ADM101UgrdSystemItemInfo(String grpId, String sysId, String sysNm,
			Double sysSeq, String admSysYn, String msgSysYn, String sysDesc,
			String mangDept, String buseoName1, String mangDept1,
			String buseoName2) {
		super();
		this.grpId = grpId;
		this.sysId = sysId;
		this.sysNm = sysNm;
		this.sysSeq = sysSeq;
		this.admSysYn = admSysYn;
		this.msgSysYn = msgSysYn;
		this.sysDesc = sysDesc;
		this.mangDept = mangDept;
		this.buseoName1 = buseoName1;
		this.mangDept1 = mangDept1;
		this.buseoName2 = buseoName2;
	}
	public String getGrpId() {
		return grpId;
	}
	public void setGrpId(String grpId) {
		this.grpId = grpId;
	}
	public String getSysId() {
		return sysId;
	}
	public void setSysId(String sysId) {
		this.sysId = sysId;
	}
	public String getSysNm() {
		return sysNm;
	}
	public void setSysNm(String sysNm) {
		this.sysNm = sysNm;
	}
	public Double getSysSeq() {
		return sysSeq;
	}
	public void setSysSeq(Double sysSeq) {
		this.sysSeq = sysSeq;
	}
	public String getAdmSysYn() {
		return admSysYn;
	}
	public void setAdmSysYn(String admSysYn) {
		this.admSysYn = admSysYn;
	}
	public String getMsgSysYn() {
		return msgSysYn;
	}
	public void setMsgSysYn(String msgSysYn) {
		this.msgSysYn = msgSysYn;
	}
	public String getSysDesc() {
		return sysDesc;
	}
	public void setSysDesc(String sysDesc) {
		this.sysDesc = sysDesc;
	}
	public String getMangDept() {
		return mangDept;
	}
	public void setMangDept(String mangDept) {
		this.mangDept = mangDept;
	}
	public String getBuseoName1() {
		return buseoName1;
	}
	public void setBuseoName1(String buseoName1) {
		this.buseoName1 = buseoName1;
	}
	public String getMangDept1() {
		return mangDept1;
	}
	public void setMangDept1(String mangDept1) {
		this.mangDept1 = mangDept1;
	}
	public String getBuseoName2() {
		return buseoName2;
	}
	public void setBuseoName2(String buseoName2) {
		this.buseoName2 = buseoName2;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/ADM101UgrdSystemItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    [Serializable]
	public class ADM101UgrdSystemItemInfo
	{
		private String _grpId;
		private String _sysId;
		private String _sysNm;
		private String _sysSeq;
		private String _admSysYn;
		private String _msgSysYn;
		private String _sysDesc;
		private String _mangDept;
		private String _buseoName1;
		private String _mangDept1;
		private String _buseoName2;
		private String _dataRowState;

		public String GrpId
		{
			get { return this._grpId; }
			set { this._grpId = value; }
		}

		public String SysId
		{
			get { return this._sysId; }
			set { this._sysId = value; }
		}

		public String SysNm
		{
			get { return this._sysNm; }
			set { this._sysNm = value; }
		}

		public String SysSeq
		{
			get { return this._sysSeq; }
			set { this._sysSeq = value; }
		}

		public String AdmSysYn
		{
			get { return this._admSysYn; }
			set { this._admSysYn = value; }
		}

		public String MsgSysYn
		{
			get { return this._msgSysYn; }
			set { this._msgSysYn = value; }
		}

		public String SysDesc
		{
			get { return this._sysDesc; }
			set { this._sysDesc = value; }
		}

		public String MangDept
		{
			get { return this._mangDept; }
			set { this._mangDept = value; }
		}

		public String BuseoName1
		{
			get { return this._buseoName1; }
			set { this._buseoName1 = value; }
		}

		public String MangDept1
		{
			get { return this._mangDept1; }
			set { this._mangDept1 = value; }
		}

		public String BuseoName2
		{
			get { return this._buseoName2; }
			set { this._buseoName2 = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public ADM101UgrdSystemItemInfo() { }

		public ADM101UgrdSystemItemInfo(String grpId, String sysId, String sysNm, String sysSeq, String admSysYn, String msgSysYn, String sysDesc, String mangDept, String buseoName1, String mangDept1, String buseoName2, String dataRowState)
		{
			this._grpId = grpId;
			this._sysId = sysId;
			this._sysNm = sysNm;
			this._sysSeq = sysSeq;
			this._admSysYn = admSysYn;
			this._msgSysYn = msgSysYn;
			this._sysDesc = sysDesc;
			this._mangDept = mangDept;
			this._buseoName1 = buseoName1;
			this._mangDept1 = mangDept1;
			this._buseoName2 = buseoName2;
			this._dataRowState = dataRowState;
		}

	}
}